File: Sismique/ClientApp/app/components/shared/reportdescription.ts
export interface IReportDescriptionProperties {
    id: number;
    reportDetail: string;
    hazard: string;
    magnitude: string;
    year: string;
    location: string;
    source: string;
    collector: string;
    buildingStories: string;
    tagging: string;
}

export class ReportDescription {
    id: number;
    reportDetail: string;
    hazard: string;
    magnitude: string;
    year: string;
    location: string;
    source: string;
    collector: string;
    buildingStories: string;
    tagging: string;

    deserialize(input: IReportDescriptionProperties): ReportDescription {
        this.id = input.id;
        this.reportDetail = input.reportDetail;
        this.hazard = input.hazard;
        this.magnitude = input.magnitude;
        this.year = input.year;
        this.location = input.location;
        this.source = input.source;
        this.collector = input.collector;
        this.buildingStories = input.buildingStories;
        this.tagging = input.tagging;
        return this;
    }
}

File: Sismique/Models/ReportDescription.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace ARIO.Models
{
    public class ReportDescription
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ID { get; set; }
        public string ReportDetail { get; set; }
        public string Hazard { get; set; }
        public string Magnitude { get; set; }
        public string Year { get; set; }
        public string Location { get; set; }
        public string Source { get; set; }
        public string Collector { get; set; }
        public string buildingStories { get; set; }
        public string tagging { get; set; }
        public Report Report { get; set; }
    }
}